v0.25.0b3 stopped to build on Windows

While v0.25.0b2 was building ok, with v0.25.0b3 I get

    linking shared-object rugged/rugged.so
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(filebuf.c.obj): In function `write_deflate':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/filebuf.c:176: undefined reference to `deflate'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(filebuf.c.obj): In function `git_filebuf_cleanup':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/filebuf.c:123: undefined reference to `deflateEnd'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(filebuf.c.obj): In function `git_filebuf_open_withsize':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/filebuf.c:318: undefined reference to `deflateInit_'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(zstream.c.obj): In function `git_zstream_init':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/zstream.c:33: undefined reference to `deflateInit_'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(zstream.c.obj): In function `git_zstream_reset':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/zstream.c:44: undefined reference to `deflateReset'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(zstream.c.obj): In function `git_zstream_get_output':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/zstream.c:100: undefined reference to `deflate'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/zstream.c:100: undefined reference to `deflate'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(zstream.c.obj): In function `git_zstream_free':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/zstream.c:39: undefined reference to `deflateEnd'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/zstream.c:39: undefined reference to `deflateEnd'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(odb_loose.c.obj): In function `finish_inflate':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:212: undefined reference to `inflate'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:214: undefined reference to `inflateEnd'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(odb_loose.c.obj): In function `read_header_loose':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:444: undefined reference to `inflateInit_'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:449: undefined reference to `inflate'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(odb_loose.c.obj): In function `start_inflate':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:201: undefined reference to `inflateInit_'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:204: undefined reference to `inflate'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(odb_loose.c.obj): In function `inflate_tail':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:292: undefined reference to `inflateEnd'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:277: undefined reference to `inflateEnd'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(odb_loose.c.obj): In function `inflate_buffer':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:245: undefined reference to `inflateInit_'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:251: undefined reference to `inflate'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/odb_loose.c:253: undefined reference to `inflateEnd'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(pack.c.obj): In function `packfile_unpack_compressed':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:869: undefined reference to `inflateInit_'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:880: undefined reference to `inflate'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:895: undefined reference to `inflateEnd'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:887: undefined reference to `inflateEnd'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(pack.c.obj): In function `git_packfile_stream_open':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:792: undefined reference to `inflateInit_'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(pack.c.obj): In function `git_packfile_stream_read':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:818: undefined reference to `inflate'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(pack.c.obj): In function `git_packfile_stream_free':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:843: undefined reference to `inflateEnd'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/pack.c:843: undefined reference to `inflateEnd'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(indexer.c.obj): In function `crc_object':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/indexer.c:246: undefined reference to `crc32'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/indexer.c:253: undefined reference to `crc32'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(indexer.c.obj): In function `hash_and_save':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/indexer.c:388: undefined reference to `crc32'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(indexer.c.obj): In function `inject_object':
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/indexer.c:706: undefined reference to `crc32'
    c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/indexer.c:714: undefined reference to `crc32'
    C:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/ext/rugged/../../vendor/libgit2/build/libgit2.a(indexer.c.obj):c:/Ruby/lib/ruby/gems/2.3.0/gems/rugged-0.25.0b3/vendor/libgit2/src/indexer.c:724: more undefined references to `crc32' follow
    collect2.exe: error: ld returned 1 exit status
    make: *** [rugged.so] Error 1

    make failed, exit code 2

--- a/ext/rugged/extconf.rb
+++ b/ext/rugged/extconf.rb
@@ -77,7 +77,7 @@ else
       # in $LIBS or the final linking stage won't pick them up
       if windows?
         $LDFLAGS << " " + "-L#{Dir.pwd}/deps/winhttp"
-        $LIBS << " -lwinhttp -lcrypt32 -lrpcrt4 -lole32"
+        $LIBS << " -lwinhttp -lcrypt32 -lrpcrt4 -lole32 -lz"
       else
         pcfile = File.join(LIBGIT2_DIR, "build", "libgit2.pc")
         $LDFLAGS << " " + `pkg-config --libs --static #{pcfile}`.strip

